script/utils/rewards_testing/TestRewardsV2.sol

```diff
@@ -0,0 +1,334 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import "./TransactionSubmitter.sol";
+import "script/utils/ExistingDeploymentParser.sol";
+
+import "@openzeppelin/contracts/token/ERC20/presets/ERC20PresetFixedSupply.sol";
+import "zeus-templates/utils/ZeusScript.sol";
+
+contract TestRewardsV2 is ZeusScript {
```

Comment:
How do we run this using Zeus?

Reply:
no just sources from env

Reply:
Shouldn't this be a script file then? : `TestRewardsV2.s.sol`

Reply:
dfc18a3d